packages/lib/getEventTypeById.ts

```diff
@@ -279,9 +279,20 @@ export default async function getEventTypeById({
   // Sets to null if no membership is found - this must mean we are in a none team event type
   const currentUserMembership = eventTypeObject.team?.members.find((el) => el.user.id === userId) ?? null;
 
+  let destinationCalendar = eventTypeObject.destinationCalendar;
+  if (!destinationCalendar) {
+    destinationCalendar = await prisma.destinationCalendar.findFirst({
+      where: {
+        userId: userId,
+        eventTypeId: null,
+      },
+    });
+  }
```

Comment:
If there is no destination calendar associated with the event type, then grab the default destination calendar